CRM/Admin/Form/MessageTemplates.php

```diff
@@ -284,6 +284,7 @@ public function postProcess() {
     else {
       // store the submitted values in an array
       $params = $this->controller->exportValues();
+      $params['is_active'] = CRM_Utils_Array::value('is_active', $params, FALSE);
```

Comment:
Thanks for the PR, just note the `value` function is [marked deprecated](https://github.com/civicrm/civicrm-core/blob/14a24788ed9762a3d02dc3e128bd5a1d354afaa4/CRM/Utils/Array.php#L50). We usually use `??` except for the situations where it's not a 1-1 swap (e.g. if $params['is_active'] = null were a potentially valid value).

But, the way we usually deal with checkbox is to use advcheckbox instead (up on line 210). It's a drop-in replacement for checkbox that always submits a value instead of the age-old wacky checkbox. If you `grep -r advcheckbox CRM` you'll see it's used plenty.

Reply:
Thanks for reviewing. I removed the above line and instead changed the element to advcheckbox as suggested.

Reply:
Great. Are you able to [squash commits](https://docs.civicrm.org/dev/en/latest/tools/git/#rebasing-interactive)? If not that's ok it's just preferred.

Reply:
OK, done.